examples/docs_beta_snippets/docs_beta_snippets/guides/data-assets/data-assets/graph_asset_decorator.py

```diff
@@ -0,0 +1,17 @@
+import dagster as dg
+
+
+@dg.graph_asset
+def complex_asset():
+    @dg.op
+    def step_one(): ...
+
+    @dg.op
+    def step_two(): ...
+
+    step_one()
```

Comment:
I think the graph needs to return the output of an op, and the ops should be dependent on each other: `return step_two(step_one())`